sqlmesh/core/snapshot/evaluator.py

```diff
@@ -1071,6 +1081,7 @@ def create(
         table_name: str,
         model: Model,
         is_table_deployable: bool,
+        is_snapshot_deployable: bool,
```

Comment:
Can we not make it a part of the public API to avoid making it more complex for users willing to implement a custom strategy. Can we hide this in `kwargs`?

Reply:
Done, it's been banished back to `kwargs`